config/config.msft.yaml

```diff
@@ -215,3 +221,11 @@ clouds:
           # lets create this MSI manually for the time being and automate soon
           # but we should use the MSI name as an input and not the resource ID of the MSI
           aroDevopsMsiId: '/subscriptions/5299e6b7-b23b-46c8-8277-dc1147807117/resourcegroups/global-shared-resources/providers/Microsoft.ManagedIdentity/userAssignedIdentities/aroint-int-public-oidc'
+
+          # Geographies
+          geographies:
```

Comment:
@geoberle  likely want to make sure this structure is how we want to hold data for each region

Reply:
no the config.schema.json describes the result after interpreting a config.yaml in a cloud/env/region. if we want to support geographies as well, we need to make that part of the config.yaml structure itself